src/components/MultiStepIndicator/MultiStepIndicator.tsx

```diff
@@ -12,32 +9,58 @@ import { SIZES, MULTI_STEP_TYPES, STEP_STATUSES, TEXT_PLACEMENTS } from "./Multi
 import { getTestId } from "../../tests/test-ids-utils";
 import { ComponentDefaultTestId } from "../../tests/constants";
 import styles from "./MultiStepIndicator.module.scss";
+import { withStaticProps, VibeComponentProps, VibeComponent, SubIcon } from "../../types";
+import { IconType } from "../Icon/IconConstants";
 
-const MultiStepIndicator = forwardRef(
+export type Step = {
```

Comment:
Is it ok, according to your standards, to export the `Step` type like that? it makes sense since it's only relevant (at least for now) to the `MultiStepIndicator` scope (story + tests)

Reply:
It's make sense. Did you see similar exports in other components? (Just to make sure we keep the API consistent as possible)

Reply:
Actually after checking it, yep, seems like it is a pattern when needed

Reply:
Add it to API_GUIDE

Reply:
https://monday.monday.com/boards/3532714909/pulses/4818659218